src/argilla/client/feedback/schemas/metadata.py

```diff
@@ -148,16 +156,66 @@ class FloatMetadataProperty(MetadataPropertySchema):
     """
 
     type: MetadataPropertyTypes = MetadataPropertyTypes.float
-    min: float
-    max: float
+    min: Optional[float] = None
+    max: Optional[float] = None
 
-    _bounds_validator = root_validator(allow_reuse=True)(validate_numeric_metadata_property_bounds)
+
+class MetadataFilterSchema(BaseModel, ABC):
+    name: str = Field(..., regex=r"^(?=.*[a-z0-9])[a-z0-9_-]+$")
+    type: MetadataPropertyTypes = Field(..., allow_mutation=False)
+
+    class Config:
+        validate_assignment = True
+        extra = Extra.forbid
+        exclude = {"type"}
+
+    @abstractproperty
+    def server_settings(self) -> Dict[str, Any]:
```

Comment:
I understand the `server_settings` purpose, but since the filter schema has no settings and they will be used as part of the query param in the URL, maybe we should change the signature:
```python
def as_server_repr(self) -> str:
# or
def to_server_schema(self) -> str:

```
```

Reply:
True, I'll change it now for the `MetadataFilters`, thanks!